### DISCUSSION TITLE
网页翻译分段出现问题，请开发者看一下是哪里出Bug了，感谢🙇‍

### DISCUSSION THREAD

**Discussion by CarolejaneCosmos** [immersive-translate/immersive-translate#1285] [Help] [answered]
### 概述 | Summary

在翻译https://www.tandfonline.com/网页内文章时会出现将参考文献作为分段的情况。
例如：https://www.tandfonline.com/doi/full/10.1080/00380768.2019.1635872
![image](https://github.com/immersive-translate/immersive-translate/assets/108039782/fc5abb66-308f-4a30-b8f4-4357d7d2c638)
麻烦大佬看一下是哪里出Bug了🙇‍

**Comment by CarolejaneCosmos** [comment]
邀请回答一下🙏 @theowenyoung

**Comment by keyboard3** [comment, marked as answer]
![image](https://github.com/immersive-translate/immersive-translate/assets/13177744/df61ffff-126c-4863-b5ef-3d2414ed32f8)
已适配，稍等2分钟刷新下页面再试下

**Reply by CarolejaneCosmos** [reply]
感谢！

**Comment by carleenmiao** [comment]
webofsecience/res.js，这个部分最后，网页不能翻译，会出错，是什么原因啊

![WechatIMG279](https://github.com/immersive-translate/immersive-translate/assets/155867361/c5c97df3-5f67-4a16-b792-f456087c451c)
